javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/resolution/ContextTest.java

```diff
@@ -825,6 +825,57 @@ void localVariablesExposedToChildWithinTryWithResourcesStatement() {
         assertOneVarExposedToChildInContextNamed(stmt, stmt.getTryBlock(), "res1");
     }
 
+    @Nested
+    class RecordPatternExprTests {
```

Comment:
Tests to ensure declarations in nested patterns are correctly exposed to children